docs/network-policy.md

```diff
@@ -0,0 +1,70 @@
+# NetworkPolicy
+Kubernetes NetworkPolicy documentation: https://kubernetes.io/docs/concepts/services-networking/network-policies/#behavior-of-to-and-from-selectors
+
+Each NetworkPolicy object consists of three sections:
+
+1. `podSelector`: a label selector that determines which pods the NetworkPolicy applies to
+2. `ingress`: determines which entities ingress traffic can be received from by the pods selected by the NetworkPolicy
+3. `egress`: determines which entities egress traffic can be sent to by pods selected by the NetworkPolicy
+
+## Unicast default-deny
+When a pod is selected by one or more NetworkPolicies it becomes isolated and all unicast ingress and egress traffic is blocked. Two global PortGroups are used to facilitate this behavior: `ingressDefaultDeny` and `egressDefaultDeny`.  Any pod selected by a NetworkPolicy in any namespace is added to these PortGroups.
+
+Two ACLs (four total) are added to each PortGroup:
+
+1. a drop policy with `priority=1000` and `direction=to-lport`
+2. an allow policy for ARP traffic with `priority=1001`, `direction=to-lport`, and `match=arp`
+
+## Multicast
+All multicast traffic is blocked by default, and must be enabled on a per-Namespace basis using an annotation on that Namespace.
```

Comment:
It is probably worth clarifying that multicast ACLs do not use NetworkPolicy at all (and maybe this file should be "OVN-Kubernetes's use of ACLs" rather than "NetworkPolicy"?)